### DISCUSSION TITLE
AppImage for Linux aarch64

### DISCUSSION THREAD

**Discussion by Aquassarim** [OrcaSlicer/OrcaSlicer#3709] [Ideas]
Hello.
I use Orca Slicer with pleasure on my x86_64 Linux PC.
For some time now, I've been trying to migrate to the Raspberry Pi, which I now find powerful enough for office use, with very limited power consumption. For example, FreeCAD and Prusa run very well on the Pi 5 under Raspberry Pi OS. I'd like to know if it would be possible to offer a Linux aarch64 version of Orca Slicer in the form of an AppImage? I have no idea of how much work this represents but it would offer support for extra printers on this platform.
Thank you!
Aqua

**Comment by bmentink** [comment]
Yep, and I would like it on my Asahi Linux running om MacBook M1
